### ISSUE TITLE
Feature request: Addition of Task filter "Kappa"

### ISSUE THREAD

**Issue by cj2tech** [TarkovTracker/TarkovTracker#422]
It would be dope to have a filter that would show "needed-to-complete" and "need-to-complete and available" for Kappa in the task tab . Below a visual to help show what I mean.

![image](https://github.com/TarkovTracker/TarkovTracker/assets/9431091/09f7a46b-e725-4cf8-a482-ff536ef4e30b)

**Comment by r-gruber** [comment]
Not sure if you know, but you can already activate/deactivate filter by non-kappa quests in the settings.

**Comment by cj2tech** [comment]
> Not sure if you know, but you can already activate/deactivate filter by non-kappa quests in the settings.

Yeah I know there is an option in the settings but it would be nice to have a option on the menu to show only kappa quest

**Comment by frmjar** [comment]
I don't think it makes much sense to have that control if someone already has the option to hide non-KAPPA quests enabled. The same information would be shown in both 'all' and 'KAPPA' filters.
The only thing that might be done is to show that filter if the user has the non-KAPPA quests filter disabled. But even so, it seems a bit redundant to me.

**Comment by cj2tech** [comment]
> I don't think it makes much sense to have that control if someone already has the option to hide non-KAPPA quests enabled. The same information would be shown in both 'all' and 'KAPPA' filters. The only thing that might be done is to show that filter if the user has the non-KAPPA quests filter disabled. But even so, it seems a bit redundant to me.

I'd admit it is a little redundant but it would also be nice if you were grinding for kappa. Also not having to goto settings everytime to see kappa quest

**Comment by DysektAI** [comment]
> It would be dope to have a filter that would show "needed-to-complete" and "need-to-complete and available" for Kappa in the task tab . Below a visual to help show what I mean.
> 
> ![image](https://github.com/TarkovTracker/TarkovTracker/assets/9431091/09f7a46b-e725-4cf8-a482-ff536ef4e30b)



## Has your issue been resolved? @cj2tech 
- **Yes** → please close this issue (use **Close issue** or **Close with comment** below).
- **No** → leave a short comment so others know it's still a problem.

> [!IMPORTANT]
> This repository and [TarkovTracker.io](https://TarkovTracker.io) are **currently unmaintained**.  
> It’s unlikely that new issues or pull requests will receive a response from the original maintainers.

### 👉 Looking for the active project?
- **Website:** [TarkovTracker.org](https://TarkovTracker.org)
- **Source code:** [tarkovtracker-org/TarkovTracker](https://github.com/tarkovtracker-org/TarkovTracker)
- **Open a new issue here:** https://github.com/tarkovtracker-org/TarkovTracker/issues/new/choose
- **Project status & changelog:** #455 

_Thanks for keeping things tidy and for using TarkovTracker!_
